diff --git a/src/bin/bat/clap_app.rs b/src/bin/bat/clap_app.rs
index adf466a..214a843 100644
--- a/src/bin/bat/clap_app.rs
+++ b/src/bin/bat/clap_app.rs
@@ -387,7 +387,7 @@ pub fn build_app(interactive_output: bool) -> Command {
                 .overrides_with("detect-color-scheme")
                 .value_name("when")
                 .value_parser(["auto", "never", "always"])
-                .default_value("auto")
+                .default_value("always")
                 .hide_default_value(true)
                 .help("Specify when to query the terminal for its colors.")
                 .long_help(
