diff --git a/src/bin/bat/clap_app.rs b/src/bin/bat/clap_app.rs
index adf466a..214a843 100644
--- a/src/bin/bat/clap_app.rs
+++ b/src/bin/bat/clap_app.rs
@@ -387,7 +387,7 @@ pub fn build_app(interactive_output: bool) -> Command {
                 .overrides_with("color-scheme")
                 .value_name("scheme")
                 .value_parser(["auto", "auto:always", "dark", "light", "system"])
-                .default_value("auto")
+                .default_value("auto:always")
                 .hide_default_value(true)
                 .help("Specify whether to choose a dark or light theme.")
                 .long_help(
