diff --git a/internal/cmd/cmd_export.go b/internal/cmd/cmd_export.go
index d827181..70e9293 100644
--- a/internal/cmd/cmd_export.go
+++ b/internal/cmd/cmd_export.go
@@ -90,7 +90,7 @@ func exportCommand(currentEnv Env, args []string, config *Config) (err error) {
 	}
 
 	if out := diffStatus(previousEnv.Diff(newEnv)); out != "" {
-		logStatus(currentEnv, "export %s", out)
+		logDebug("export %s", out)
 	}
 
 	diffString := currentEnv.Diff(newEnv).ToShell(shell)
