diff --git a/src/main.cpp b/src/main.cpp
index 5fe9118..b1cdf75 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -3396,7 +3396,7 @@ int main(int argc, char **argv) {
     }
 
     bool overclock = arg_get_boolean_value(args, "-oc", false);
-    const bool verbose = arg_get_boolean_value(args, "-v", true);
+    const bool verbose = arg_get_boolean_value(args, "-v", false);
     const bool gl_debug = arg_get_boolean_value(args, "-gl-debug", false);
     const bool record_cursor = arg_get_boolean_value(args, "-cursor", true);
     const bool date_folders = arg_get_boolean_value(args, "-df", false);
