diff --git a/src/blocks/kdeconnect.rs b/src/blocks/kdeconnect.rs
index ee14688d..70820ca5 100644
--- a/src/blocks/kdeconnect.rs
+++ b/src/blocks/kdeconnect.rs
@@ -117,6 +117,8 @@ pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
                     if battery_state {
                         widget.state = if charging {
                             State::Good
+                        } else if level <= 0 {
+                            State::Idle
                         } else if level <= config.bat_critical {
                             State::Critical
                         } else if level <= config.bat_info {
