diff --git a/src/blocks/uptime.rs b/src/blocks/uptime.rs
index 7402d3f2..dd55d0c4 100644
--- a/src/blocks/uptime.rs
+++ b/src/blocks/uptime.rs
@@ -72,7 +72,12 @@ pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
             format!("{minutes}m {seconds}s")
         };
 
-        let mut widget = Widget::new().with_format(format.clone());
+
+        let mut widget = Widget::new().with_format(format.clone()).with_state(if weeks > 0 {
+            State::Warning
+        } else {
+            State::Idle
+        });
         widget.set_values(map! {
           "icon" => Value::icon(api.get_icon("uptime")?),
           "text" => Value::text(text)
