diff --git a/kitty/desktop.c b/kitty/desktop.c
index 97618f4a..18be7057 100644
--- a/kitty/desktop.c
+++ b/kitty/desktop.c
@@ -153,6 +153,7 @@ play_canberra_sound(const char *which_sound, const char *event_id, bool is_path,
         canberra_ctx, 0,
         which_type, which_sound,
         "event.description", event_id,
+        "canberra.xdg-theme.name", "honk",
         "media.role", media_role,
         NULL
     );
