diff --git a/core/definitions.units b/core/definitions.units
index 29e3b53..4d76b8a 100644
--- a/core/definitions.units
+++ b/core/definitions.units
@@ -4582,6 +4582,7 @@ Mcf                     1000 ft^3    # not million cubic feet [18]
 kp                      kilopond
 kpm                     kp meter
 Wh                      W hour
+Ah                      A hour
 hph                     hp hour
 plf                     lb / foot    # pounds per linear foot
 
